<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Flipper</class>
 <widget class="QWidget" name="Flipper">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>444</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Flipper</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="m_renameButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Rename</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="m_keepInRamCheckbox">
       <property name="toolTip">
        <string>If checked, image files dropped into this window will be read from disk as needed rather than cached in memory.</string>
       </property>
       <property name="text">
        <string>Always store images in RAM</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="m_playbackButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>(scrubbing)</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="m_fpsLimitLabel">
       <property name="text">
        <string>FPS limit:</string>
       </property>
       <property name="buddy">
        <cstring>m_fpsLimitEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="m_fpsLimitEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="m_frameIndexLabel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Frame:</string>
       </property>
       <property name="buddy">
        <cstring>m_frameIndexSpinner</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="m_frameIndexSpinner">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximum">
        <number>999999</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSlider" name="m_frameIndexSlider">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="maximum">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBelow</enum>
     </property>
     <property name="tickInterval">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="m_frameListbox">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="m_loadingGroupbox">
     <property name="title">
      <string>Loading images...</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="m_loadingCancelButton">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="m_loadingProgressbar">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>m_renameButton</tabstop>
  <tabstop>m_keepInRamCheckbox</tabstop>
  <tabstop>m_playbackButton</tabstop>
  <tabstop>m_fpsLimitEdit</tabstop>
  <tabstop>m_frameIndexSpinner</tabstop>
  <tabstop>m_frameIndexSlider</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>m_playbackButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>Flipper</receiver>
   <slot>playbackButtonClicked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>45</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>235</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_frameIndexSlider</sender>
   <signal>sliderPressed()</signal>
   <receiver>Flipper</receiver>
   <slot>frameIndexSliderPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>235</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>235</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_frameIndexSlider</sender>
   <signal>sliderReleased()</signal>
   <receiver>Flipper</receiver>
   <slot>frameIndexSliderReleased()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>235</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>235</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_frameIndexSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Flipper</receiver>
   <slot>setFrameIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>235</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>235</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_frameIndexSpinner</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Flipper</receiver>
   <slot>setFrameIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>434</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>235</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_renameButton</sender>
   <signal>clicked()</signal>
   <receiver>Flipper</receiver>
   <slot>renameButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>379</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_keepInRamCheckbox</sender>
   <signal>toggled(bool)</signal>
   <receiver>Flipper</receiver>
   <slot>alwaysStoreImagesInRamToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>645</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>379</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_frameListbox</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>Flipper</receiver>
   <slot>setFrameIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>191</x>
     <y>256</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_fpsLimitEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>Flipper</receiver>
   <slot>fpsLimitEdited(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>190</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>playbackButtonClicked(bool)</slot>
  <slot>frameIndexSliderPressed()</slot>
  <slot>frameIndexSliderReleased()</slot>
  <slot>setFrameIndex(int)</slot>
  <slot>renameButtonClicked()</slot>
  <slot>alwaysStoreImagesInRamToggled(bool)</slot>
  <slot>fpsLimitEdited(QString)</slot>
 </slots>
</ui>
